lms/djangoapps/courseware/tests/test_courses.py

```diff
@@ -424,3 +426,30 @@ def test_repeated_course_module_instantiation(self, loops, default_store, course
                 for section in chapter.get_children():
                     for item in section.get_children():
                         self.assertTrue(item.graded)
+
+
+@attr(shard=1)
+class TestGetCourseChapters(ModuleStoreTestCase):
+    """
+    Tests for the `get_course_chapters` function.
+    """
+
+    def test_get_non_existant_course(self):
+        """
+        Test non-existant course returns empty list.
+        """
+        self.assertEqual(get_course_chapters(None), [])
+        # build a fake key
+        fake_course_key = CourseKey.from_string('course-v1:FakeOrg+CN1+CR-FALLNEVER1')
+        self.assertEqual(get_course_chapters(fake_course_key), [])
+
+    def test_get_chapters(self):
+        """
+        Test get_course_chapters returns expected result.
+        """
+        course = CourseFactory()
+        ItemFactory(parent=course, category='chapter')
+        self.assertEqual(
+            get_course_chapters(course.location.course_key),
+            [unicode(child) for child in course.children]
+        )
```

Comment:
Can we put a couple of chapters in here and have a separate check that there are in fact two chapters that spit out? This test would pass if none of the publishing actually happened (i.e. if ItemFactory doesn't properly save).